src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/index.tsx

```diff
@@ -1,39 +1,40 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { ToolbarDropdown, usePlateEditorState } from '@udecode/plate';
-import React, { useCallback, useState } from 'react';
+import { usePlateEditorState } from '@udecode/plate';
+import React, { useState } from 'react';
 import { useRichTextEditorContext } from '../../../context/RichTextEditorContext';
 import { defaultTextStyles } from '../TextStyles/defaultTextStyles';
-import { TextStyles, textStyleTitle } from '../TextStyles';
+import { textStyleTitle } from '../TextStyles';
 import { DropdownItem } from './DropdownItem';
 import { DropdownTrigger } from './DropdownTrigger';
 import { TextStyleDropdownProps } from './types';
+import { DesignTokens } from '@components/RichTextEditor/types';
+import { useTextStyleDropdown } from './useTextStyleDropdown';
 
 export const TextStyleDropdown = ({ editorId, textStyles = defaultTextStyles }: TextStyleDropdownProps) => {
-    const [open, setOpen] = useState(false);
+    const [triggerElement, setTriggerElement] = useState<HTMLButtonElement | null>(null);
```

Comment:
Would suggest to put this two states into `useTextStyleDropdown `  and try to pass editor as props, because TextStyleButton gets this prop from plate